mycroft/skills/twitter/vocab/en-us/twitterKeyword.voc

```diff
@@ -0,0 +1,3 @@
+tweet
+twitter
+post to twitter
```

Comment:
We probably do not need **post to twitter** since we have **twitter** as a keyword.